.pylintrc

```diff
@@ -1,3 +1,4 @@
 [MESSAGES CONTROL]
 disable = too-many-instance-attributes,
-          too-many-arguments
+          too-many-arguments,
+          duplicate-code
```

Comment:
What happened here? :( 

Can it be a per-file exception instead of a general one?

Reply:
Opm and OperatorSDK look very similar. I couldn't manage to make the exception where it happens.

Reply:
:sad:

Reply:
https://github.com/PyCQA/pylint/issues/214